ladybug_geometry/geometry2d/mesh.py

```diff
@@ -290,11 +294,18 @@ def remove_vertices(self, pattern):
                 The length of this list must match the number of this mesh's vertices.
 
         Returns:
-            new_mesh: A mesh where the vertices have been removed according
+            A list with two elements.
```

Comment:
Technically, this is a tuple and not a list because of Python tuple unpacking.

Reply:
Fixed!